azurelinuxagent/ga/exthandlers.py

```diff
@@ -392,6 +392,8 @@ def handle_enable(self, ext_handler_i):
            old_ext_handler_i.version_gt(ext_handler_i):
             msg = "Downgrade is not allowed. Skipping install and enable."
```

Comment:
Is this message accurate based on your change?

Reply:
Yes. This block hits a return statement at line 399. The install and enable logic is from lines 401 to 420 in this function.